### DISCUSSION TITLE
Is it possible to use createStyles with static class names?

### DISCUSSION THREAD

**Discussion by albertbronsky** [mantinedev/mantine#752] [Q&A] [answered]
Can I use `createStyles` to apply styling to static class names (I want to avoid loading a separate `.css` file for that)?

**Comment by albertbronsky** [comment, marked as answer]
I reckon that the only possible way to achieve this is by using [Global component](https://mantine.dev/theming/create-styles/#global-styles)?

**Reply by rtivital** [reply]
Yes, use Global component
